wikimedia/wikipedia-ios
# Define in article announcement user experience for iOS app

https://phabricator.wikimedia.org/T235539

![cards](https://user-images.githubusercontent.com/9299317/68361650-a6dfec80-00f2-11ea-9272-4c0ab419e7ee.png)

Testing:

1. Check out https://gerrit.wikimedia.org/r/#/c/mediawiki/services/wikifeeds/+/545071/, run locally
2. Set scheme to Local
3. Set the correct cookie - https://github.com/wikimedia/wikipedia-ios/blob/3080578435ccd75fb504e2278ee1893811fa259f/WMF%20Framework/WMFAnnouncementsFetcher.m#L36
I tested `US`(feed) and `GB` (in article)
